packages/communication-react/stable-review/communication-react.api.md

```diff
@@ -1805,6 +1805,7 @@ export type MessageThreadProps = {
     onSendMessage?: (messageId: string) => Promise<void>;
     disableEditing?: boolean;
     strings?: Partial<MessageThreadStrings>;
+    dateTimeFormat?: (messageDate: Date) => string;
```

Comment:
I am thinking of a more generic way of this function - what if customer wants to further customized their timeStamp? What if they want to hover over time stamp and show some special popup? what if they want to print a img in timestamp?
There could be another option like onRenderTimeStamp?: (messageDate: Date) => React.Component to open all these possibilities (especially in component layer, then we might be able to set more restriction in composite level)

I would like to involve @anjulgarg to this discussion, cause he's the only expert on custom data injection in our team

Reply:
Lets just only focus on strings for now and we can look into a modification at the component level if devs want to extremely customize this area